parm/wave/wave_gfs.buoys

```diff
@@ -63,7 +67,7 @@ $ Gulf of Alaska (AG) Spectral data (4) near S/SW Alaska Anchorage (8)
   -152.230  59.050 '46079     '   4.9   DAT  NDBC      45
   -152.233  59.049 '46105     '   2.0   DAT  NDBC      45
   -147.992  59.925 '46107     '   2.0   DAT  NDBC      45
-  -165.475  64.473 '46265     '   2.0   DAT  NDBC      45
+  -165.446  64.489 '46265     '   2.0   DAT  NDBC      45
```

Comment:
This has nothing to do with the COM refactor.  Can we pull out these changes to another PR so this isn't hidden in a huge PR?  I'm also curious/confused about the changes to the content of this file.

Reply:
I copied the v16 buoys file (which for v16 was in fix). I needed the full buoys file in order for the boundary points jobs to run because the pruned version did not have any. When I initially looked, I thought it was only additions, but now it seems there were also a handful of coordinate changes (buoy drift?)

Reply:
After a quick chat with Jessica, we are going to pull the buoy changes out as a separate PR to commit before this one once she checks on a few things.

Reply:
New buoy file and the other fixes for boundary point jobs pulled out into #1465